core/modules/admin_bar/js/admin_bar.js

```diff
@@ -197,8 +197,18 @@ Backdrop.adminBar.behaviors.resizeCollapse = function (context, settings, $admin
 
     $adminBar.trigger('beforeResize');
 
-    menuWidth = $menu.width();
-    extraWidth = $extra.width();
+    if ($menu.length) {
+      menuWidth = $menu.width();
+    }
+    else {
+      menuWidth = 0;
+    }
+    if ($extra.length) {
+      extraWidth = $extra.width();
+    }
+    else {
+      extraWidth = 0;
```

Comment:
I'm going to suggest a more succinct and idiomatic JS way to do this - you see this very often, including in Backdrop js files. Replace all 3 new `if` blocks with this:
```js
extraWidth = $extra.width() || 0;
```
(Of course change the variable names when dealing with $menu). Because `$extra.width()` is `undefined`, then the `||` (or) will assign the second part (0).

This can be applied to the other ifs here.

Reply:
Slick. Will do.